kali_iam.tf

```diff
@@ -40,3 +48,23 @@ resource "aws_iam_role_policy_attachment" "efs_mount_policy_attachment_kali" {
   role       = aws_iam_role.kali_instance_role.id
   policy_arn = aws_iam_policy.efs_mount_policy.arn
 }
+
+################################
+# Define the role policies below
+################################
+
+# Allow the Kali instance to assume the necessary roles to perform its
+# function.
+data "aws_iam_policy_document" "kali_assume_delegated_role_policy_doc" {
+  statement {
+    actions = [
+      "sts:AssumeRole",
+      "sts:TagSession",
+    ]
+    effect = "Allow"
+    resources = [
+      aws_iam_role.guacamole_parameterstorereadonly_role.arn,
```

Comment:
If the VNC username is no longer pulled from SSM I don't think we need to give this permission anymore.

```suggestion
```

Reply:
Please see commit 7896da0.